sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -21,7 +21,13 @@ This product is fully supported and enabled by default as of {product-title} 4.1
 [id="sandboxed-containers-1-3-bug-fixes"]
 == Bug fixes
 
+* Previously, {sandboxed-containers-first} deployed a security context constraint (SCC) on startup which enforced the use of an `osc_monitor.process` custom SELinux type that was not available on machine config operator (MCO) pods. This caused the MCO pod to change to a `CrashLoopBackOff` state and cluster upgrades to fail. With this release, {sandboxed-containers-first} deploys the SCC when creating the `KataConfig` CR and no longer enforces using the `osc_monitor.process`. (https://issues.redhat.com/browse/KATA-1373[*KATA-1373*])
 
+* Previously, when creating the `KataConfig` CR and observing the pod status under the `openshift-sandboxed-containers-operator` namespace, a huge number of restarts for monitor pods was shown. The monitor pods use a specific SELinux policy that was installed as part of the `sandboxed-containers` extension installation. The monitor pod was created immediately, however the SELinux policy was not yet available, which resulted in a pod creation error, followed by a pod restart.
++
+With this release, the SELinux policy is available when the monitor pod is created, and the monitor pod transitions to a `Running` state immediately. (https://issues.redhat.com/browse/KATA-1338[*KATA-1338*])
+
+* Previously, when uninstalling the {sandboxed-containers-operator}, the `sandboxed-containers-operator-scc` custom resource was not deleted. With this release, all resources are deleted when uninstalling the {sandboxed-containers-operator}. (https://issues.redhat.com/browse/KATA-1569[*KATA-1569*])
```

Comment:
Nit picking: the SCC custom resource is certainly deleted now, but I wouldn't pretend all the resources created by OSC are deleted.